<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedgerEditWidget</class>
 <widget class="QWidget" name="LedgerEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit ledger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="nameChecked">
       <property name="text">
        <string>Hide Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="yearComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>&amp;Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showButton">
         <property name="text">
          <string>S&amp;how</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printButton">
         <property name="text">
          <string>&amp;Print</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
